### PR TITLE
Initial state for accessibility

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: apps/accessibility/src/Accessibility.vue [hunk 1]
```diff
@@ -26,30 +26,36 @@
 <script>
 import ItemPreview from './components/ItemPreview'
 import axios from 'nextcloud-axios'
+import { generateUrl } from 'nextcloud-router'
```
Comment (ChristophWurst) [inline apps/accessibility/src/Accessibility.vue]: *cough* https://www.npmjs.com/package/@nextcloud/router *cough*

File: apps/accessibility/src/Accessibility.vue [hunk 2]
```diff
@@ -26,30 +26,36 @@
 <script>
 import ItemPreview from './components/ItemPreview'
 import axios from 'nextcloud-axios'
```
Comment (ChristophWurst) [inline apps/accessibility/src/Accessibility.vue]: *cough* https://www.npmjs.com/package/@nextcloud/axios *cough*

File: apps/accessibility/src/main.js [hunk 3]
```diff
@@ -1,12 +1,20 @@
 import Vue from 'vue'
+import { loadState } from 'nextcloud-initial-state'
```
Comment (ChristophWurst) [inline apps/accessibility/src/main.js]: really? https://www.npmjs.com/package/@nextcloud/initial-state

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Comment by rullzer** [comment]
need a rebase :)

**Comment by skjnldsv** [comment]
> need a rebase :)

:rocket:

**Review by ChristophWurst — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Pull request merged**